datafusion/src/optimizer/simplify_expressions.rs

```diff
@@ -930,16 +826,16 @@ mod tests {
         let expr_plus = binary_expr(null.clone(), Operator::Divide, null.clone());
         let expr_eq = null;
 
-        assert_eq!(simplify(&expr_plus), expr_eq);
+        assert_eq!(simplify(expr_plus), expr_eq);
     }
 
     #[test]
-    fn test_simplify_do_not_simplify_arithmetic_expr() {
```

Comment:
not sure why this test was here -- now that constant propagation is implemented, these cases do indeed work